Makefile

```diff
@@ -134,11 +134,12 @@ distclean: clean
 build-docs:
 	@cd docs && \
 	while read p; do \
-		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
+		(git checkout $${p} && npm install && VUEPRESS_BASE="/gaia/$${p}/" npm run build) ; \
```

Comment:
Not sure about adding this `gaia` prefix. Instead, we should add a file named `CNAME` to the `gh-pages` branch with the domain we will be using (`hub.cosmos.network`).

Reply:
I know it wont work under `cosmos.github.io` domain, but will under `hub.cosmos.network` which is what we want.